sdk/eventhub/event-hubs/src/eventHubReceiver.ts

```diff
@@ -477,7 +477,7 @@ export class EventHubReceiver extends LinkEntity {
           operation: () => this._init(options),
           connectionId: this._context.connectionId,
           operationType: RetryOperationType.receiverLink,
-          times: Constants.defaultConnectionRetryAttempts,
+          times: Constants.defaultMaxRetriesForConnection,
```

Comment:
Update `times` -> `maxRetries` in retry config before passing it to `retry` method.